cmd/ctl/pkg/check/version/checker/fromcrd.go

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Does it make sense for this command to be part of `kubectl cert-manager version`? I would expect the same behavior as kubectl:

```
Client Version:
Server Version:
```

Reply:
Makes sense, although please notice that this version lib is also in the `/pkg` folder.
I think I'll move my server-version detection logic back to that folder too, so its easier to use in other parts of the codebase too.

Reply:
I merged the implementations